src/post_process/m_checker.fpp

```diff
@@ -111,6 +111,21 @@ contains
         @:PROHIBIT(any(omega_wrt) .and. fd_order == dflt_int, "fd_order must be set for omega_wrt")
     end subroutine s_check_inputs_vorticity
 
+    !> Checks constraints on fft_wrt
+    impure subroutine s_check_inputs_fft
+        integer :: num_procs_x, num_procs_y, num_procs_z
+
+        @:PROHIBIT(fft_wrt .and. file_per_process, "Turn off file_per_process with fft_wrt")
+        @:PROHIBIT(fft_wrt .and. (n == 0 .or. p == 0), "FFT WRT only in 3D")
+        @:PROHIBIT(fft_wrt .and. (MOD(m_glb+1,2) == 1 .or. MOD(n_glb+1,2) == 1 .or. MOD(p_glb+1,2) == 1), "FFT WRT requires global dimensions divisible by 2")
+        num_procs_x = (m_glb + 1)/(m + 1)
```

Comment:
i think this should be removed? we shouldn't automatically adjust the number of processors for the user? what do you think